src/run.ts

```diff
@@ -69,6 +81,20 @@ async function runBreaking(): Promise<null|Error> {
             message: 'an against was not provided'
         };
     }
+    const username = core.getInput('buf_input_https_username');
+    if (username === '') {
+        return {
+            message: 'github username not provided'
```

Comment:
We should update the error message so that it matches the configuration key (or something else along those lines).
```suggestion
            message: 'buf_input_https_username not provided'
```

Reply:
Oh yeah, that's fair, good call :D